### PR TITLE
new status page

### PR DESCRIPTION
**Pull request by cmcfadzean**
status page similar to this: 
https://thumbprint.design/components/overview/
Screen Shot 2020-03-03 at 8 57 19 PM

### CODE REVIEW

File: app/assets/stylesheets/sage_docs/_status_table.scss [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+      padding-right: 0;
+    }   
+  }
+  td {
+    padding: sage-spacing(xs) sage-spacing(sm);
+    vertical-align: top;
+    &:nth-child(1) {
+      font-weight: sage-font-weight(semibold);
+      padding-left: 0;
+      width: 10000px;
```
Comment (teenwolfblitzer) [inline app/assets/stylesheets/sage_docs/_status_table.scss:20]: I'd like to see us move away from the pattern of using large fixed widths for spacing in the codebase. If necessary, we could instead set the `tr` to be a flex container and adjust the `flex-basis` accordingly.

File: app/assets/stylesheets/sage_docs/_status_table.scss [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    }
+    &:nth-child(2) {
+      border-bottom: sage-border();
+      width: 4000px;
```
Comment (teenwolfblitzer) [inline app/assets/stylesheets/sage_docs/_status_table.scss:24]: Same here as above with the fixed width value.

File: app/views/sage/pages/status.html.erb [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+        <li class="quick-links__item">
+          <a href="#<%= object[:title] %>" class="quick-links__link"><%= object[:title].titleize %></a>
+        </li>
+      <% end %>
+    </ul>
+  </div>
+<% end %>
+<div class="sage-panel">
+  <h3>Elements</h3>
+  <table class="status-table">
```
Comment (teenwolfblitzer) [inline app/views/sage/pages/status.html.erb:30]: We can address this in a follow-up (Chris and I have some ideas for tweaks and making the table into a component), but the mobile view is breaking outside of the panel:

status-table-mobile

**Review by teenwolfblitzer — APPROVED** [review]
👍 This is looking great! I've added a few things to consider that can all be addressed in a follow-up.

File: app/assets/stylesheets/sage_docs/_status_table.scss (hunk 1 above)
Comment (cmcfadzean) [reply app/assets/stylesheets/sage_docs/_status_table.scss:20]: Love it. I think this gets us moving forward but lets put in issue into the backlog to revisit this.

File: app/views/sage/application/_status_key.html.erb [hunk 4]
```diff
@@ -0,0 +1,22 @@
+<div class="status-key__wrapper">
```
Comment (philschanely) [inline app/views/sage/application/_status_key.html.erb:1]: For cleaner BEM I'd suggest this change to `status-key-wrapper`.

File: app/assets/stylesheets/sage_docs/_status_key.scss [hunk 5]
```diff
@@ -0,0 +1,42 @@
+.status-key {
+  margin: 0 sage-spacing();
+  display: flex;
+  align-items: center;
+  &:first-of-type {
+    margin-left: 0;
+  }
+  &__wrapper {
```
Comment (philschanely) [inline app/assets/stylesheets/sage_docs/_status_key.scss:8]: Per comment below, this may need would also be adjusted to keep in sync.

**Review by philschanely — COMMENTED** [review]
A few suggestions for cleaner BEM.

File: app/views/sage/application/_status_key.html.erb (hunk 4 above)
Comment (cmcfadzean) [reply app/views/sage/application/_status_key.html.erb:1]: Would you not consider it to be an element of the status-key block? Interested to hear more.

**Pull request merged**